src/Me/components/RichList/RichItem.js

```diff
@@ -0,0 +1,208 @@
+import PropTypes from 'prop-types';
+import React, { useRef } from 'react';
+import styled from 'styled-components';
+import { CSSTransition } from 'react-transition-group';
+
+const style = (
+  <style>{`
```

Comment:
Why using plain css but not Styled?

Reply:
Because  `CSSTransition` relies on a base class name to then derive the other class names based on transition-state so it was easiest to use global style.
I'm sure it possible to use Styled but I'm new to this library and was short on time.
Add it to my technical debt, please.

Reply:
What do you think?

https://codesandbox.io/s/agitated-wu-4pbzb?file=/src/ToggleableContent.js

Reply:
tbh this was my first approach but I wanted to try out `CSSTransition` 😅
I guess this component doesn't need the extra flexibility. I'll change it later.